Documentation/contributing/testing/ci.rst

```diff
@@ -141,6 +141,8 @@ illustrating which subset of tests the job runs.
 +---------------------------------------------------------------------------------------------------------+-------------------+--------------------+
 | `Cilium-Pr-Ginkgo-Test-k8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-k8s/>`_                | test-missed-k8s   | No                 |
 +---------------------------------------------------------------------------------------------------------+-------------------+--------------------+
+| `Cilium-PR-Ginkgo-Tests-Validated <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/>`_   | restart-ginkgo    | Yes                |
```

Comment:
@mrostecki Could you drop this commit from the backport PR please? I removed the label in the original PR. We've discussed this a bit today in #launchpad and establish that it's probably not worth it because we would need to update the whole table. Sorry about this!

Reply:
I'm sorry for not following the #launchpad channel! I will drop it.

Reply:
Np. Plus, I posted in the #launchpad channel after you had already opened this PR.